eos_data_distribution/avahi_monitor.py

```diff
@@ -33,6 +33,8 @@
 from eos_data_distribution.MDNS import ServiceDiscovery
```

Comment:
This commit is missing a commit message explaining why it’s a good idea to limit the number of peers.